packages/webapp/src/utils/helpers.js

```diff
@@ -3,10 +3,22 @@ import constants from './constants.js'
 import registerTaskList from './register-task-list.js'
 import validator from 'email-validator'
 
-const validateDate = (payload, ID, desc) => {
-  const day = payload[`${ID}-day`]
-  const month = payload[`${ID}-month`]
+const isoDateFormat = 'YYYY-MM-DD'
+
+const parsePayload = (payload, ID) => {
+  const day = (payload[`${ID}-day`] && payload[`${ID}-day`].length === 1) ? payload[`${ID}-day`].padStart(2, 0) : payload[`${ID}-day`]
```

Comment:
Should `.padStart` pad with a string? Probably makes no difference

Reply:
I agree it will probably make no difference but I will make it explicitly a string for clarity